keras_nlp/src/models/retinanet/anchor_generator_test.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import parameterized
+from keras import ops
+
+from keras_nlp.src.bounding_box.converters import convert_format
+from keras_nlp.src.models.retinanet.anchor_generator import Anchor
+from keras_nlp.src.tests.test_case import TestCase
+
+
+class AnchorTest(TestCase):
+    @parameterized.parameters(
+        # Single scale anchor
+        (
```

Comment:
Let's modify these so they format a little more readable/condensed. This might work?

```
        ("yxyx", 5, 5, 1, [1.0], 2.0, [64, 64])
        + (
            {
                "P5": [
                    [-16.0, -16.0, 48.0, 48.0],
                    [-16.0, 16.0, 48.0, 80.0],
                    [16.0, -16.0, 80.0, 48.0],
                    [16.0, 16.0, 80.0, 80.0],
                ]
            },
        ),
```

Reply:
Done